fvtest/threadtest/abortTest.cpp

```diff
@@ -167,7 +167,7 @@ sleepingMain(void *arg)
 
 	EXPECT_EQ(rc, J9THREAD_PRIORITY_INTERRUPTED);
 
-	return rc;
+	return 0;
```

Comment:
It's a thread entry-point, it's signature must be
```C++
typedef int (*functionPoint)(void *);
```
In this case, we really don't care about the return.